test/contrib/postgres_test.py

```diff
@@ -124,6 +126,10 @@ def test_bulk_complete(self, mock_connect):
         ])
         self.assertFalse(task.complete())
 
+    def test_override_port(self):
+        output = DummyPostgresQueryWithPort(date=datetime.datetime(1991, 3, 24)).output()
+        self.assertEquals(output.port, 1234)
```

Comment:
Could we also add a test for the `host:port` parsing? I'm not seeing one here

Reply:
Should that be part of this PR? I think it might fit better to test https://github.com/spotify/luigi/blob/master/luigi/contrib/postgres.py#L101 instead, because that's where parsing happens.

Reply:
I'm fine with that :+1:

Reply:
I gave it a try. PTAL again. https://github.com/spotify/luigi/pull/2627/commits/22a6566043155f43c803e6694c0d56d60fea7201